### DISCUSSION TITLE
WebGL image support

### DISCUSSION THREAD

**Discussion by cbarbaz** [GitbookIO/community#245] [Feature Requests]
### What's your feature/idea?

Hey ! I'm wondering if there are any plans to support WebGL images in GitBook docs soon? Thanks!

### What's your desired outcome?

Integrating WebGL support would result in lighter-weight pages and is becoming a must-have feature in today's digital landscape.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by addisonschultz** [comment]
Nice feature request - I think this might be something you could do using our integration platform! https://developer.gitbook.com/

Natively, we use https://iframely.com/domains to embed urls in GitBook - but for the ones that aren't supported, creating an integration is the next way to go. 

Do you have any examples of what you'd like to include?

**Comment by cbarbaz** [comment]
Hi Addison,
I would just like to be able to upload WebGL images instead of png or jpg.

**Reply by addisonschultz** [reply]
Hi @cbarbaz, I'm a bit confused at what you're referring to exactly - do you mean .webp files? If so, these should be supported already!

If not, and can share an example of a WebGL image I'd be happy to look into this more!

**Reply by cbarbaz** [reply]
Hi Addison,
You're right, I meant .webp, sorry for confusing you!
I've just tested again to add a .webp image to an article and it worked, indeed! (I don't know why it didn't last time i tried...). I will thus close this discussion.
Thanks a lot for your help!
